### PR TITLE
new: philippines-travel-level-map

### PR DESCRIPTION
**Pull request by ejcenteno**
This pr adds the trending `philippines-travel-level-map` repo to the list of `pinoy-made` apps repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -370,3 +371,10 @@ Perform telescope.nvim functions in a selected directory
 - [Github](https://github.com/princejoogie/dir-telescope.nvim)
 - Author: [@princejoogie](https://github.com/princejoogie)
 - Twitter: [@princecaarlo](https://twitter.com/princecaarlo)
+
+## philippines-travel-level-map
```
Comment (jofftiquez) [inline README.md:375]: Hi, thanks for this. Please change the title to Philippines Travel Level Map

File: README.md [hunk 2]
```diff
@@ -370,3 +371,10 @@ Perform telescope.nvim functions in a selected directory
...
+
+Mark the places you've visited and see which other more you wanna go to!
```
Comment (jofftiquez) [inline README.md:377]: Also, add the link the website itself

**Review by jofftiquez — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ejcenteno) [reply README.md]: Hello, I have changed the title. Thanks

File: README.md (hunk 2 above)
Comment (ejcenteno) [reply README.md:377]: Done sir, thank you.

File: README.md [hunk 3]
```diff
@@ -370,3 +371,12 @@ Perform telescope.nvim functions in a selected directory
 - [Github](https://github.com/princejoogie/dir-telescope.nvim)
 - Author: [@princejoogie](https://github.com/princejoogie)
 - Twitter: [@princecaarlo](https://twitter.com/princecaarlo)
+
+## Philippines Travel Level Map
+
+Mark the places you've visited and see which other more you wanna go to!
+Visit [https://www.my-philippines-travel-level.com](https://www.my-philippines-travel-level.com).
```
Comment (jofftiquez) [inline README.md:378]: Please follow this same pattern 
image

**Review by jofftiquez — CHANGES_REQUESTED** [review]

**Review by jofftiquez — APPROVED** [review]
Looks good, thank

**Pull request merged**